rpcd/playbooks/roles/rpc_support/defaults/main.yml

```diff
@@ -70,6 +70,12 @@ holland_packages:
   - mariadb-client
   - xtrabackup
 
+holland_venv_enabled: True
+holland_venv_tag: "{{ rpc_release }}"
+holland_venv_bin: "/openstack/venvs/holland-{{ holland_venv_tag }}/bin"
```

Comment:
This probably got in an older commit, but it would be good to move the "rpc_release" down to this line, and remove "holland_venv_tag" altogether.